client/state/data-layer/wpcom/sites/test/utils.js

```diff
@@ -166,7 +165,6 @@ describe( 'utility functions', () => {
 
 			handleWriteCommentFailure( { dispatch, getState }, { siteId: 2916284, postId: 1010 } );
 
-			expect( dispatch ).to.have.been.calledOnce;
```

Comment:
Dispatch is now called twice here (once for the notice and once for COMMENTS_WRITE_ERROR). I didn't think an `expect( dispatch ).to.have.been.calledTwice` added much value to the test.